ravenframework/SupervisedLearning/DynamicModeDecompositionControl.py

```diff
@@ -397,6 +478,66 @@ def writeXML(self, writeTo, targets = None, skip = None):
                    "matrixShape":",".join(str(x) for x in np.shape(self.__Ctilde[smp, :, :]))}
         writeTo.addVector("Ctilde","realization",valDict, root=targNode, attrs=attributeDict)
 
+  def getSolutionMetadata(self):
+    """
+      Get solution metadata
+    """
```

Comment:
Please update the docstring here.

Reply:
There is a return from this function. Please add it in the docstring

Reply:
done